spec/abci++/abci++_tmint_expected_behavior.md

```diff
@@ -56,22 +59,23 @@ consensus-exec      = (inf)consensus-height
 consensus-height    = *consensus-round decide commit
```

Comment:
"decide" is not defined.
You probably want to leave it here, and define as follows:
```
decide = begin-block  *deliver-txs end-block 
```